content/docs/querying/functions.md

```diff
@@ -40,6 +40,12 @@ by sample value.
 `ceil(v instant-vector)` rounds the sample values of all elements in `v` up to
 the nearest integer.
 
+## `changes()`
+
+For each input time series, `changes(v range-vector)` returns the number of
+times its value has changed within the provided time range as an instant
+vector.
```

Comment:
If protobuf preserves the representation, then yes - don't know the internals.
We can certainly add a notice later if there are people asking questions about it.